packages/headless/src/database/mongodb/user/index.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@nestjs/common';
+import { UserEntity } from 'src/entity/user';
+import { IUserDB } from 'src/modules/user/repository/user.db.interface';
+import { UserModel } from './user.model';
+
+@Injectable()
+export class UserAuthDB implements IUserDB {
+  async save(body: UserEntity): Promise<UserEntity | null> {
+    const user = await UserModel.create(body);
+    delete user?.password;
+    return user;
+  }
+  async findOneForLogin(
+    query: Record<string, string>,
+  ): Promise<UserEntity | null> {
+    const user = await UserModel.findOne(query).lean();
```

Comment:
Instead of 2 line of code, we can return it on the first line.
like 
`return await UserModel.findOne(query).lean();`